common/lib/xmodule/xmodule/js/spec/video/completion_spec.js

```diff
@@ -51,6 +52,15 @@
             expect(state.completionHandler.markCompletion).not.toHaveBeenCalled();
         });
 
+        it('does not call the completion api on the LMS when the video is loaded but not seen', function() {
+            var verticalStudentView;
+            spyOn(window, 'VerticalStudentView').and.callThrough();
+            // eslint-disable-next-line no-unused-vars
```

Comment:
What is the reason for this comment?

Reply:
This is the eslint error I was talking about. When a new verticalStudentView object is made, it looks through the DOM for any .vert elements, checks if they are completable upon viewing, and, if yes, sends a call to the completion API to mark them complete. The bug was happening because verticalStudentView was not catching that a video xblock is not completable upon viewing, so I want to be able to make a verticalStudentView object and verify that it doesn't send an ajax call for a video xblock. There's no reason to use the verticalStudentView object, since I'm just making it for the side-effect.

I could do something like: ```new window.VerticalStudentView(null, '#video_example');``` and not assign it to any variable, but then I get a [different](https://eslint.org/docs/rules/no-new) eslint error.

Spying on ```window.VerticalStudentView``` does not count as using the verticalStudentView variable because the former is a spy on a function, and the second is the function itself.

Reply:
I see. I am fine with the eslint ignore comments. But I think we can change the name of the not used variable `verticalStudentView` to be something like `notUsedCreatedForTest` or something like that.